recipes-kernel/linux/files/uos_rt.cfg

```diff
@@ -0,0 +1,17 @@
+CONFIG_ACRN_VIRTIO_DEVICES=y
+CONFIG_HYPER_DMABUF_ACRN=y
+CONFIG_ACRN=y
+CONFIG_PREEMPT_COUNT=y
```

Comment:
Preempt-RT releated configuration can be skipped by fixing  LINUX_KERNEL_TYPE = "preempt-rt".
We will fix this.

Reply:
We fixed preempt-RT releated configuration with https://github.com/intel/meta-acrn/commit/e375f02ecda59ff26d638a306def2a4629755cdb. Please give a try and let us know what else should be enabled?

Reply:
OK, we will just add rest configs

Reply:
Just keep the additional configuration only in uos_rt.cfg from uos.cfg. Apply both linux-intel-rt-acrn-uos_4.19.bb

SRC_URI_append = "  file://uos.cfg"
SRC_URI_append = "  file://uos-rt.cfg"

Reply:
OK, got it.